### DISCUSSION TITLE
Adding specific quantity of a part as installed item(s)

### DISCUSSION THREAD

**Discussion by palagap** [inventree/InvenTree#5397] [Q&A]
Hi there, 

We noticed an issue when trying to install a specific quantity of a part to a build. We’ve replicated this using the InvenTree demo:

1. We start with an item, such as the Blue Chair.

![Screenshot 1](https://github.com/inventree/InvenTree/assets/40808444/0039ac3b-95f5-4499-ae4f-37315dece47a)

2. We attempt to install a Stock Item (e.g. Screws).

![Screenshot 2](https://github.com/inventree/InvenTree/assets/40808444/57581849-d370-489a-b560-ff19d0db5dde)

3. When selecting the stock item, though, we are only given the option to select the full quantity of screws, rather than being able to enter the desired quantity.

![Screenshot 3](https://github.com/inventree/InvenTree/assets/40808444/29475c51-5f13-44da-b879-62647fdc8457)

4. For instance, the BOM shows that only 5 screws are needed.

![Screenshot 6](https://github.com/inventree/InvenTree/assets/40808444/c0e4b0d0-603b-4d34-a4fe-3e8526340f84)

5. However, we have to install all 1300 of them in this case.

![Screenshot 5](https://github.com/inventree/InvenTree/assets/40808444/09be762a-8bc6-4ac5-8489-1b46cde4e90b)

Is there a way to avoid this issue and only install the correct number of screws by manually entering the desired number of screws to install? Thanks.

**Comment by SchrodingersGat** [comment]
@palagap currently you cannot specify the quantity when "installing" items - I suppose that this is because I imagined it would only ever be used for single, serialized items.

I'll create an issue from this discussion, I can see that it would be a good addition.

**Reply by palagap** [reply]
great, thank you.
